configure

```diff
@@ -179,6 +179,7 @@ my @specs = # Array of array-refs, each inner array is an option group which
     'prefix=s', 'Installation prefix (none)',
     'install-origin-relative!', 'Install with RPATH relative to $ORIGIN (no)',
     'workspace=s', 'Custom MPC workspace file to copy and use (Use a builtin one)',
+    'coverage!', 'Enable code coverage (no)',
```

Comment:
Does this need to be its own option or could it just be `--features=<something>`?

Reply:
We could do it that way.  What are the long-term maintenance tradeoffs between `--coverage` and `--features=dds_non_coverage=0`?

Reply:
I don't expect this to be used all that much outside of the maintainers/contributors -- I'd rather keep the configure script simpler and not have to know about every single way the configuration can be tweaked through MPC (or gnuace or config.h...).